utils/treeStructure.py

```diff
@@ -97,6 +103,10 @@ def setDefaults(self, options, time):
         self.slot[0] = options.slot
         self.utime[0] = time
 
+        from gempython.gemplotting.mapping.chamberInfo import chamber_config
+
+        self.detName[0] = chamber_config[(options.shelf,options.slot,options.gtx)]
```

Comment:
Here is likely where we could consider thinking of how to pull the name out of the DB

Reply:
I get the error

```
cx_Oracle.DatabaseError: ORA-01017: invalid username/password; logon denied
```

when I try to access the DB in my virtual environment. Is there a trick to access the DB in a virtual environment?